lib/ft817/ft817.cpp

```diff
@@ -169,7 +169,9 @@ void FT817::setMode(byte mode)
 		flushBuffer();
 		buffer[0] = mode;
 		buffer[4] = CAT_MODE_SET;
-		getByte();
+		// missing a sendCmd here...
+		sendCmd();
+		getByte();     

```

Comment:
Added the missing sendCmd() to make this work.

If I can add comments to fixes in the pull request like this, is it best practice to not comment in the code like I did here?

Reply:
tks